docs/HowTo/Use/node_network_migration.md

```diff
@@ -0,0 +1,107 @@
+---
+description: Migrate node to a new IP network
+---
+
+# Migrate node to a new IP network
+
+When changing the network configuration of a node, it's recommended to follow specific steps so the new configuration it's applied and the consensus protocol continues to work as expected.
+
+!!!important
+
+    Before starting the migration, create a backup of your data directory of all Nodes.
```

Comment:
Is this something documented somewhere else that we could link to?

Reply:
there is https://docs.goquorum.consensys.net/en/stable/HowTo/Use/import-export/#node-backup-export, but in this case, I don't want to stop the Nodes if is not necessary.

also, based on https://docs.goquorum.consensys.net/en/stable/HowTo/Use/import-export/#special-consensus-considerations, it won't include RAFT data.

any other docs that might have that?

Reply:
I'll add a link to the Geth documentation: https://geth.ethereum.org/docs/install-and-build/backup-restore#data-directory